### PR TITLE
Feature/42 sync suite feature tests

### PR DESCRIPTION
**Pull request by knysh**
# PR Details

## Related Issue

#42 

## How Has This Been Tested

syncSuite.spec.ts

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: e2e/specs/suite/syncSuite.spec.ts [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  });
+
+  using(notEditorExamples, (user, description) => {
+    describe(`${description} role:`, () => {
+      beforeAll(async () => {
+        await logIn.logInAs(user.user_name, user.password);
+        await projectHelper.openProject();
+      });
+
+      it(`Sync suite is disabled for role ${description}`, async () => {
```
Comment (VladislavKostyukevich) [inline e2e/specs/suite/syncSuite.spec.ts]: rename to `Sync suite is not present for role ${description}`

File: e2e/specs/suite/syncSuite.spec.ts [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+  afterAll(async () => {
+    await projectHelper.dispose();
+  });
+
+  using(notEditorExamples, (user, description) => {
+    describe(`${description} role:`, () => {
+      beforeAll(async () => {
+        await logIn.logInAs(user.user_name, user.password);
+        await projectHelper.openProject();
```
Comment (VladislavKostyukevich) [inline e2e/specs/suite/syncSuite.spec.ts]: return last statement

File: e2e/specs/suite/syncSuite.spec.ts [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+      admin: users.admin,
+      localAdmin: users.localAdmin,
+      localManager: users.localManager,
+      localEngineer: users.localEngineer,
+      viewer: users.viewer
+    });
+  });
+
+  afterAll(async () => {
+    await projectHelper.dispose();
```
Comment (VladislavKostyukevich) [inline e2e/specs/suite/syncSuite.spec.ts]: return last statement

File: e2e/specs/suite/syncSuite.spec.ts [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+      let syncTestRuns: TestResult[];
+
+      beforeAll(async () => {
+        await logIn.logInAs(users.admin.user_name, users.admin.password);
+        await projectHelper.openProject();
+        notSyncTestRun = await projectHelper.importer.executeCucumberImport(testSuite, [cucumberImport], [builds.filenames[0]]);
+        syncTestRuns = await projectHelper.importer.executeCucumberImport(
+          testSuite, [syncImport, syncImport], [builds.filenames[1], builds.filenames[2]]);
+        await logIn.logInAs(user.user_name, user.password);
+        await projectHelper.openProject();
```
Comment (VladislavKostyukevich) [inline e2e/specs/suite/syncSuite.spec.ts]: return last statement

File: e2e/specs/suite/syncSuite.spec.ts [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+const noSyncTest = 'Test Feature with all results: All passed';
+const syncTest = 'Test Feature with all results: step failed';
+const syncAndRemoveTest = 'Test Feature with all results: Step skipped';
+
+describe('Sync Test Suite', () => {
+  const projectHelper: ProjectHelper = new ProjectHelper();
+  const builds = projectHelper.generateBuilds(3);
+
+  beforeAll(async () => {
+    await projectHelper.init({
```
Comment (VladislavKostyukevich) [inline e2e/specs/suite/syncSuite.spec.ts]: return last statement

File: e2e/specs/suite/syncSuite.spec.ts (hunk 1 above)
Comment (knysh) [reply e2e/specs/suite/syncSuite.spec.ts]: fixed

File: e2e/specs/suite/syncSuite.spec.ts (hunk 2 above)
Comment (knysh) [reply e2e/specs/suite/syncSuite.spec.ts]: fixed

File: e2e/specs/suite/syncSuite.spec.ts (hunk 3 above)
Comment (knysh) [reply e2e/specs/suite/syncSuite.spec.ts]: fixed

File: e2e/specs/suite/syncSuite.spec.ts (hunk 4 above)
Comment (knysh) [reply e2e/specs/suite/syncSuite.spec.ts]: fixed

File: e2e/specs/suite/syncSuite.spec.ts (hunk 5 above)
Comment (knysh) [reply e2e/specs/suite/syncSuite.spec.ts]: fixed

**Pull request merged**